src/test/s3/commands/presignedURL.test.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import { copyUrl, presignedURLCommand, promptTime } from '../../../s3/commands/presignedURL'
+import { S3BucketNode } from '../../../s3/explorer/s3BucketNode'
+import { S3FileNode } from '../../../s3/explorer/s3FileNode'
+import { S3Node } from '../../../s3/explorer/s3Nodes'
+import { Bucket, S3Client } from '../../../shared/clients/s3Client'
+import { Window } from '../../../shared/vscode/window'
+import { FakeEnv } from '../../shared/vscode/fakeEnv'
+import { anything, instance, mock, when } from '../../utilities/mockito'
+
+describe('presignedURLCommand', function () {
+    const bucketName = 'bucket-name'
+    const key = 'file.jpg'
+    const fileName = 'file.jpg'
+    const testUrl = 'https://presigned-url.com'
+
+    let env: FakeEnv
+    let node: S3FileNode
+    let s3: S3Client
+    let bucketNode: S3BucketNode
+    let window: Window
+
+    beforeEach(function () {
+        s3 = mock()
+        window = mock()
+        env = new FakeEnv()
+
+        const bucket: Bucket = { name: bucketName, region: 'region', arn: 'arn' }
+        bucketNode = new S3BucketNode(bucket, {} as S3Node, instance(s3))
+        node = new S3FileNode(bucket, { name: key, key: key, arn: 'arn' }, bucketNode, instance(s3))
+    })
+
+    it("returns the user's input time", async function () {
```

Comment:
This does not look like it tests anything meaningful. Consider removing this test.